src/app/features/lp-form/reducer.ts

```diff
@@ -710,29 +724,29 @@ export const slice = createSlice({
         updateLeverageOptionsAfterGetInfoPostLp(state);
         validateUserInputAndUpdateSubmitButton(state);
       })
-      .addCase(setSignerAndPositionForAMMThunk.pending, (state) => {
-        state.position.value = null;
-        state.position.status = 'pending';
+      .addCase(setSignerAndPositionsForAMMThunk.pending, (state) => {
```

Comment:
if would move this on start of function. Basically if no amm, return early and don't do anything. It is not possible to have positions without AMM